### PR TITLE
Add CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
**Description:**
In scope of this pull request we add CODE_OF_CONDUCT
**Related issue:**

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at opensource+actions/setup-go@@github.com. All
```
Comment (IvanZosimov) [inline CODE_OF_CONDUCT.md:58]:
```suggestion
reported by contacting the project team at opensource+actions/setup-java@github.com. All
```

**Review by marko-zivic-93 — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (vsafonkin) [inline CODE_OF_CONDUCT.md:58]: setup-go -> setup-java

Comment (dmitry-shibanov) [reply CODE_OF_CONDUCT.md]: Done. Thank you.

Comment (dmitry-shibanov) [reply CODE_OF_CONDUCT.md]: Done. Thank you.

**Review by vsafonkin — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Pull request merged**